datafusion/expr-common/src/signature.rs

```diff
@@ -74,6 +74,8 @@ pub enum Volatility {
 /// adds a cast such as `cos(CAST int_column AS DOUBLE)` during planning.
 ///
 /// # Data Types
+///
+/// ## Timestamps
```

Comment:
should there be one empty line after a header?

Reply:
Good call -- fixed